lib/logstash/filters/xml.rb

```diff
@@ -35,29 +35,30 @@ class LogStash::Filters::Xml < LogStash::Filters::Base
   #
   # XML in the value of the source field will be expanded into a
   # datastructure in the `target` field.
-  # Note: if the `target` field already exists, it will be overridden
-  # Required
+  # Note: if the `target` field already exists, it will be overridden.
+  # Required.
   config :target, :validate => :string
```

Comment:
Thank you for you PR, I think we have a small inconsistency here, not related to your code.

The `target` option is only used when `store_xml` is set to **true** (the default) therefore required in that case, the register method should validate that. Maybe we can explain when this settings is used in the comment?

Create a PR for it, https://github.com/logstash-plugins/logstash-filter-xml/issues/9